airflow/models/dag.py

```diff
@@ -1747,7 +1747,8 @@ def create_dagrun(
 
         :meta private:
         """
-        logical_date = timezone.coerce_datetime(logical_date)
+        if logical_date is not None:
+            logical_date = timezone.coerce_datetime(logical_date)
```

Comment:
```suggestion
        logical_date = timezone.coerce_datetime(logical_date)
```

This function handles None on its own

Reply:
Thanks, I Implemented this change.